overall fix of utf8 handling

lodwicz opened the pull request:
a few changes of the script `ini2html.pl` to make it work on modern Perl5; all changes go into a new file, `ini2html.plx` to preserve the archive.

lodwicz committed:
fixing ini2html.pl

fixing deprecated pragmas and utf8 support

lodwicz committed:
prevent double utf conversion

jarnosz commented:
addresse issue https://github.com/Portable-Keyboard-Layout/Portable-Keyboard-Layout/issues/23#issue-1989079403

presidento commented:
IMHO Perl is practically a dead language. It would be better to rewrite it to Python. Even PHP is better more widely used than Perl. When I created it long time ago it was a reasonable choice, but nowadays the Perl community almost disappeared and still declining.

On the other hand, I'm not against make it working, just please update the original file, do not keep a non-working one just for preserving the archive - Git is already in charge of this.

lodwicz committed:
fix utf8 handling for newer perl5

removed deprecated pragma encoding; fix utf8 handling in ini file and output html file.

lodwicz committed:
delete ini2html.plx

jarnosz commented:
did as requested.
